requirements.txt

```diff
@@ -1,28 +1,28 @@
 # Autogenerated by 'make bs'
 -i https://pypi.org/simple
-bracex==2.2.1
-click==8.1.2
+bracex==2.2.1; python_version >= '3.6'
```

Comment:
These diffs come along when you run `make bs` on linux. I don't know if this file is just stale or if this is a conflict with Mac.

Reply:
I think it was just stale. I think this new format used to not be compatible with ReadTheDocs, but seems to work now, so go for it.